actors/builtin/miner/partition_state.go

```diff
@@ -401,7 +401,7 @@ func (p *Partition) PopExpiredSectors(store adt.Store, until abi.ChainEpoch, qua
 	if p.Faults, err = bitfield.SubtractBitField(p.Faults, expiredSectors); err != nil {
 		return nil, err
 	}
-	p.LivePower = p.LivePower.Sub(popped.ActivePower)
+	p.LivePower = p.LivePower.Sub(popped.ActivePower.Add(popped.FaultyPower))
```

Comment:
@anorth double checking that this makes sense.

Reply:
It does